string program_id
---
int32 status
string message
niryo_robot_programs_manager/Program program
